python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -484,6 +484,8 @@ class RunRecord(NamedTuple):
     pipeline_run: PipelineRun
```

Comment:
Could we avoid the OSS / internal breakage by switching this to inherit from `namedtuple`?